Collections/Game.java

```diff
@@ -0,0 +1,53 @@
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Comparator;
+
+enum Enum {
+    GENRE1, GENRE2, GENRE3
+}
+
+public class Game {
+    private String title;
+    private Enum genre;
+    private Double price;
+    private Integer copiesSold;
+
+    public Double getPrice() {
+        return price;
+    }
+
+    public Game(String title, Enum genre, Double price, Integer copiesSold) {
+        this.title = title;
+        this.genre = genre;
+        this.price = price;
+        this.copiesSold = copiesSold;
+    }
+
+    @Override
+    public String toString() {
+        return "Game{" +
+                "title='" + title + '\'' +
+                ", genre=" + genre +
+                ", price=" + price +
+                ", copiesSold=" + copiesSold +
+                '}';
+    }
+
+    public static void main(String[] args) {
+        Game gameOne = new Game("Game N1", Enum.GENRE1, 15.99, 100);

```

Comment:
You could create a separate method to initialise data to make your code more readable

Reply:
How it could look like?

Reply:
`private List<Game> createGames() { //Your code to create and return list of games}`